webapp/src/utils/jira_issue_metadata.tsx

```diff
@@ -264,6 +276,10 @@ export function isLabelField(field: JiraField | FilterField): boolean {
     return field.schema.system === 'labels' || field.schema.custom === 'com.atlassian.jira.plugin.system.customfieldtypes:labels';
 }
 
+export function isCommentVisibilityField(field: JiraField | FilterField): boolean {
+    return field.key === 'commentVisibility';
```

Comment:
You are using the same string key in 280 and 215, I think you can use a constant for that key and use it in both places to void future problems.